modules/core/shared/src/main/scala/scaladex/core/service/WebDatabase.scala

```diff
@@ -7,11 +7,13 @@ import scala.concurrent.Future
 
 import scaladex.core.model.Artifact
 import scaladex.core.model.ArtifactDependency
+import scaladex.core.model.GithubInfo
+import scaladex.core.model.GithubResponse
 import scaladex.core.model.Project
 import scaladex.core.model.UserState
 
 trait WebDatabase {
-  def insertArtifact(artifact: Artifact, dependencies: Seq[ArtifactDependency], time: Instant): Future[Unit]
+  def insertArtifact(artifact: Artifact, dependencies: Seq[ArtifactDependency], time: Instant): Future[Boolean]
```

Comment:
Can you add a comment to explain what this boolean means?

Reply:
Not my favourite trick.. I will add a comment